client/src/components/EditPost.js

```diff
@@ -0,0 +1,34 @@
+import React, {useEffect} from 'react';
+import PostEntry from './PostEntry'
+
+function EditPost(props) {
+  let values = {
+    title: null,
+    imageLink: null,
+    link: null,
+    subject: null,
+  }
```

Comment:
Since you are managing state with these variables, they should be declared with `useState`:

```suggestion
  let [values, setValues] = useState({
    title: null,
    imageLink: null,
    link: null,
    subject: null,
  });
```

Reply:
Also, I would rename this to something more self-explanatory, like `postData`.